fission/src/ui/modals/common/ConfirmModal.tsx

```diff
@@ -0,0 +1,53 @@
+import { Stack, Typography } from "@mui/material"
+import type React from "react"
+import { useEffect } from "react"
+import { useUIContext } from "@/ui/helpers/UIProviderHelpers"
+import type { ModalImplProps } from "@/ui/components/Modal"
+
+export type ConfirmModalCustomProps = {
+    title?: string
+    message: string
+    acceptText?: string
+    cancelText?: string
+    allowClickAway?: boolean
+    onConfirm?: () => void
+    onCancel?: () => void
+}
```

Comment:
A lot of this is redundant and already included in the regular (not custom) props. You could delete `title`, `acceptText`, `cancelText`, `allowClickAway` and just import them from `modal.prop` instead of `modal.props.custom`. For some reason there are some type issues with the modal/panel opening functions that I must have overlooked that make the linter complain about passing in the custom callbacks, but once those are resolved you will be able to use the regular `onAccept` and `onCancel` callback props instead of `onConfirm` and `onCancel`.